cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java

```diff
@@ -28,36 +31,60 @@ public ServicesOperations(DefaultCloudFoundryOperations cloudFoundryOperations)
      */
     public List<ServiceBean> getAll() {
         List<ServiceInstanceSummary> services = this.cloudFoundryOperations
-                .services()
-                .listInstances()
-                .collectList()
-                .block();
+            .services()
+            .listInstances()
+            .collectList()
+            .block();
 
         if (services == null) {
             services = new LinkedList<>();
         }
 
-        // create a list of special bean data objects, as the summaries cannot be serialized directly
+        // create a list of special bean data objects, as the summaries cannot be
+        // serialized directly
         List<ServiceBean> beans = new ArrayList<>(services.size());
         for (ServiceInstanceSummary serviceInstanceSummary : services) {
-            beans.add(new ServiceBean(serviceInstanceSummary));
+
+            GetServiceInstanceRequest getServiceInstanceRequest = GetServiceInstanceRequest.builder()
+                .name(serviceInstanceSummary.getName())
+                .build();
+            ServiceInstance serviceInstance = this.cloudFoundryOperations
+                .services()
+                .getInstance(getServiceInstanceRequest)
+                .block();
+
+            ServiceBean serviceBean = new ServiceBean(serviceInstanceSummary);
+            
+            if (serviceInstance.getLastOperation() == null
+                || serviceInstance.getLastOperation().isEmpty()) {
+                serviceBean
+                    .setLastOperation("");
```

Comment:
unncessary line change for .setLastOperation

Reply:
it's not really unimportant. if serviceInstance.getLastOperation () is null, then serviceInstance.getStatus () is also null. So we used to prevent output "null null"

Reply:
I just meant that you can write it in one line: serviceBean.setLastOperation("");

Reply:
Done